### ISSUE TITLE
Non load balanced web service type

### ISSUE THREAD

**Issue by shaicoleman** [aws/copilot-cli#1356]
Please add an option to create a web service that is not load balanced. A load balancer adds additional costs, and it's not always needed.

This was possible with fargatecli

**Comment by efekarakus** [comment]
Hi @shaicoleman !

If we implemented the user experience described in this issue: https://github.com/aws/copilot-cli/issues/1304#issuecomment-677813496 would it satisfy your use case?

**Comment by efekarakus** [comment]
Oh or did I misunderstand the request and you'd like to query against the public IP address of a Fargate task?

**Comment by shaicoleman** [comment]
Yes, it would just output the URLs of endpoints with the public IP of each container.
